spring-cloud-data-shell/src/test/java/org/springframework/cloud/data/shell/AbstractShellIntegrationTest.java

```diff
@@ -147,30 +149,20 @@ protected StreamCommandTemplate stream() {
 	}
 
 	/**
-	 * Return a unique random stream name for testing.
-	 *
-	 * @param name name to use as part of stream name
-	 * @return unique random stream name
-	 */
-	protected String generateStreamName(String name) {
-		return (name == null) ? generateUniqueName() : generateUniqueName(name);
-	}
-
-	/**
-	 * Return a unique random stream name for testing.
+	 * Return a {@link TaskCommandTemplate} for issuing shell based
+	 * task commands.
 	 *
-	 * @return unique random stream name
+	 * @return template for issuing task commands
 	 */
-	protected String generateStreamName() {
-		return generateStreamName(null);
+	protected TaskCommandTemplate task() {
+		return new TaskCommandTemplate(cloudDataShell);
 	}
-
 	// Util methods
-	private String generateUniqueName(String name) {
+	protected String generateUniqueName(String name) {
```

Comment:
since they are now protected instead of private, these methods should have JavaDoc (similar to what was deleted for the previous `generateStreamName()` method).